### ISSUE TITLE
Accessibility: No keyboard support for drawing shapes with @mapbox/mapbox-gl-draw

### ISSUE THREAD

**Issue by mbondyra** [mapbox/mapbox-gl-draw#1410]
Hi Mapbox team! 👋

At Elastic, we're using the `@mapbox/mapbox-gl-draw` library to let users draw shapes on the map, which we then use for spatial filtering in Kibana.

However, we've noticed that the current drawing functionality seems to be accessible only via mouse or touch, and we couldn’t find a way to draw using keyboard controls.

Could you confirm whether keyboard-based shape drawing is currently supported in mapbox-gl-draw?
If not, is this something that’s on the roadmap?

We’d love to make this feature accessible to all users. Here's the corresponding Kibana issue for context:
🔗 https://github.com/elastic/kibana/issues/215841

Thanks so much for your work on this library!

**Comment by AviFriedlander-OPS** [comment]
I would like to bump this request and agree that it would be a really beneficial feature. Currently, we have been unable to make applications which use this library compliant with [WCAG 2.1.1 accessibility criteria](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html).
